### PR TITLE
Copy changes (marketing review)

### PR DESCRIPTION
**Pull request by hthillman**
Also removed non-comfyUI option from filter view and moved Pipelines above Streams in the UI

### CODE REVIEW

File: apps/app/app/explore/page.tsx [hunk 1]
```diff
@@ -63,14 +63,16 @@ const Filter = () => {
...
+        <Label>ComfyUI supported</Label>
         <div className="flex flex-col space-y-2 mt-2">
           <div className="flex items-center space-x-2 bg-sidebar p-3 ">
             <Checkbox className="" id="comfyui" />
             <Label className="text-foreground/80 font-normal" htmlFor="comfyui">
-              Comfy UI supported
+              ComfyUI supported
             </Label>
           </div>
+          {/*
```
Comment (hthillman) [inline apps/app/app/explore/page.tsx:74]: commented this out @thomshutt

File: apps/app/components/sidebar.tsx [hunk 2]
```diff
@@ -87,31 +88,31 @@ export const GlobalSidebar = ({ children }: GlobalSidebarProperties) => {
     settings: NavItem[];
     footer: NavItem[];
   } = {
-    streams: [
+    pipelines: [
```
Comment (hthillman) [inline apps/app/components/sidebar.tsx:91]: flipped streams and pipelines in the sidebar @thomshutt

File: apps/app/components/welcome/index.tsx [hunk 3]
```diff
@@ -50,9 +50,10 @@ const Intro = () => {
 
   return (
     <div className="col-span-1">
-      <h3 className="font-medium text-lg">
-        Welcome {name ? `back, ${name}` : "to Livepeer"}
-      </h3>
+      <h2 className="font-medium text-lg">
```
Comment (hthillman) [inline apps/app/components/welcome/index.tsx:53]: adjusted this ternary @thomshutt

File: apps/app/components/welcome/index.tsx [hunk 4]
```diff
@@ -75,6 +76,15 @@ const Intro = () => {
         >
           What is a pipeline? <ArrowTopRightIcon className="ml-1 h-4 w-4" />
         </Button>
+        <Button
```
Comment (hthillman) [inline apps/app/components/welcome/index.tsx:79]: added button @thomshutt

File: apps/app/app/explore/page.tsx (hunk 1 above)
Comment (thomshutt) [reply apps/app/app/explore/page.tsx:74]: Why did you add the braces?

**Comment by thomshutt** [comment]
Should we also add some copy here while we're making these changes? 
image

**Review by thomshutt — APPROVED** [review]
Overall LGTM! I clicked around the UI and it all still seemed to work as expected (and had your copy changes applied)

**Comment by hthillman** [comment]
> Should we also add some copy here while we're making these changes? image

I'll handle that in another PR

**Pull request merged**